frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/DistributionField.tsx

```diff
@@ -290,6 +290,47 @@ export function DistributionField(props: {
                 />
             </>
         )
+    } else if (props.cluster.distribution.upgradeInfo?.posthookDidNotRun) {
+        // CURATOR POSTHOOK JOB DID NOT RUN
+        return (
+            <>
+                <div>{props.cluster?.distribution.displayVersion}</div>
+                <AcmInlineStatus
+                    type={StatusType.danger}
+                    status={t('upgrade.upgradefailed', {
+                        version: props.cluster?.consoleURL
+                            ? ''
+                            : props.cluster?.distribution.upgradeInfo.desiredVersion,
+                    })}
+                    popover={
+                        props.cluster?.consoleURL
+                            ? {
+                                  headerContent: t('upgrade.upgradefailed', {
+                                      version: props.cluster?.distribution.upgradeInfo.desiredVersion,
+                                  }),
+                                  bodyContent: t('Upgrade posthook was not run.', {
+                                      clusterName: props.cluster?.name,
+                                      version: props.cluster?.distribution.upgradeInfo.desiredVersion,
+                                  }),
+                                  footerContent: (
+                                      <a
+                                          href={
+                                              `${props.cluster?.consoleURL}/k8s/ns/${props.cluster.namespace}` +
+                                              '/cluster.open-cluster-management.io~v1beta1~ClusterCurator/' +
+                                              `${props.cluster.name}`
+                                          }
+                                          target="_blank"
+                                          rel="noreferrer"
+                                      >
+                                          {t('upgrade.upgrading.link')} <ExternalLinkAltIcon />
```

Comment:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/42188127/207914729-d16ccb5c-8ba8-442f-9845-dec682543b45.png">

Notice the underline between the text and icon? This pattern has crept in throughout our code and I've been meaning to open a consistency issue to make sure this is done consistently throughout our code. I think the better pattern is like this:
```
<Link to={linkTarget} target={'_blank'}>
    <Button variant="link" icon={<ExternalLinkAltIcon />} iconPosition="right" isInline>
        {showInstallPrompt ? t('Install the operator') : t('View installed operators')}
    </Button>
</Link>
```
You may need to adjust this for your needs. Also note now that we run exclusively inside OpenShift console, we can often use the `Link` component from react-router-dom. We only need to use `a` if we're going to an external site or different cluster.